src/Uno.UWP/Globalization/NumberFormatting/NumeralSystemTranslator.cs

```diff
@@ -0,0 +1,167 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Windows.Globalization.NumberFormatting
+{
+	public partial class NumeralSystemTranslator
+	{
+		private static Dictionary<string, char[]> s_numeralSystemToDigitsDictionary =
```

Comment:
Please use `_underscoreNamingConvention` for private members (and maybe this one can be made `readonly` too?)

Reply:
We use the naming like `s_something` when porting from WinUI, where we tend to retain the C++ conventions used there, but here we just use plain underscore